x-pack/plugins/monitoring/common/http_api/elasticsearch_settings/get_elasticsearch_settings_nodes.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as rt from 'io-ts';
+
+export const getElasticsearchSettingsNodesResponsePayloadRT = rt.type({
+  // TODO: add payload entries
```

Comment:
does this refer to the same type from https://github.com/elastic/kibana/blob/main/x-pack/plugins/monitoring/server/lib/elasticsearch_settings/nodes.test.ts#L77?

Reply:
As mentioned above, these are explicitly left as placeholders for the client-side TS conversion.